docs/cosmos-hub/installation.md

```diff
@@ -20,7 +20,7 @@ source ~/.bash_profile
 
 ### Install the binaries
 
-Next, let's install the latest version of Gaia. Here we'll use the `master` branch, which contains the latest stable release.
+Next, let's install the latest version of Gaia. Here we'll use the `develop` branch, which contains the latest stable release.
```

Comment:
I think we want to point users to the latest release branch. So maybe link to https://github.com/cosmos/cosmos-sdk/releases?